app/lib/search/index_simple.dart

```diff
@@ -378,22 +381,42 @@ class SimplePackageIndex implements PackageIndex {
   }
 
   _TextResults _searchText(Set<String> packages, String text) {
+    final sw = Stopwatch()..start();
```

Comment:
Can't we print something like `_logger.info('[pub-search-query-duration]: Search query took ${sw.elapsedMilliseconds} ms');`

That would be pretty cool to use for a custom metrics :)

Reply:
We could have that in `search/handlers.dart`, however, we already log that as part of the standard URL logging, e.g. every `/search?q=` request on the `search` service is just that.

Reply:
(However, if it is easier for the metrics processing, I'll add it in `search/handlers.dart`...)